### DISCUSSION TITLE
Не работает загрузочный образ ISO

### DISCUSSION THREAD

**Discussion by prologonline** [mikopbx/Core#718] [General ( Обо всём )]
При попытке записать загрузочный ISO образ с помощью рекомендуемой ImageUSB отдельный ПК Intel NUK говорит что не найден загрузочный носитель. Записал с помощью UltraISO - теперь начинает загружаться, но vmlinuz не найден(точно фразу не помню).
Как-то давно ставил уже MikoPBX из образа на VDS - вроде всё было нормально.
Есть какие-то ограничения на размер флэшки например или еще что? Пробовал на Transcend 128Gb и SanDisk 32Gb - результат один и тот-же.
Что делать и как правильно записать этот чертов образ и установить таки MikoPBX на отдельную машину?

**Comment by boffart** [comment]
Нужно пробовать записывать в "режиме dd", пробуйте другие утилиты. "рекомендованные" ведут себя странно, один релиз нормально пишут, другой релиз не работает. 

Вот тут коллеги "руфусом в режиме dd" успешно записали; 
https://t.me/mikopbx_dev/14106

**Reply by prologonline** [reply]
Спасибо! Только так и сработало.
